.docker/ci/Dockerfile

```
@@ -0,0 +1,52 @@
+ARG ROS_DISTRO=noetic
+FROM ros:${ROS_DISTRO}-ros-base
+
+ARG MUJOCO_VERSIONS=[2.3.6]
+
+ENV TERM xterm
+ENV MUJOCO_DIR=/root/mujoco/2.3.6
+
+# Setup temporary ROS workspace
+WORKDIR /root/ws_mujoco
+
+# Copy MuJoCo ROS sources from docker context
+COPY . src/mujoco_ros_pkgs
+
+# Combined commands for less stages
+RUN \
+    apt-get -q update && \
+    apt-get -q -y dist-upgrade && \
+    #
+    # Base dependencies
+    apt-get -q install --no-install-recommends -y \
+        wget curl git sudo python3-wstool \
+        # Build tools
+        python3-catkin-tools \
+        clang clang-format-10 clang-tidy clang-tools \
+        ccache && \
+    #
+    # Install MuJoCo Versions
+    MUJOCO_VERSIONS=${MUJOCO_VERSIONS} ./src/mujoco_ros_pkgs/.docker/ci/mujoco_installer.sh >> mujoco_install.log && \
```

You shouldn't write a log file in the docker image:
```suggestion
    MUJOCO_VERSIONS=${MUJOCO_VERSIONS} ./src/mujoco_ros_pkgs/.docker/ci/mujoco_installer.sh && \
```